### ISSUE TITLE
AAC recorded in IOS doesn't play in IE11 browser

### ISSUE THREAD

**Issue by jooejooe** [jsierles/react-native-audio#313]
### Tell us which versions you are using ( you can see the versions in your package.json ):

react-native-audio :  4.1.3
react-native : 0.55.3
react :  16.3.1
using react-native-sound ? Yes。version is 0.10.9 

### Expected behaviour :

I am recording an audio with params below:
`AudioRecorder.prepareRecordingAtPath(action.payload, {
        SampleRate: 22050,
        Channels: 1,
        AudioQuality: 'Low',
        AudioEncoding: 'aac'
      });`
I am getting audio.aac and play it on my website.
I expect the audio file will play well in Chrome,firefox and IE11.

### Actual behaviour
The file play well in chrome and firefox
The file does not work in IE11

### Steps to reproduce

i use html5 audio to play the aac file
`<audio controls="controls" src="/file/download?fileid=4028d0f66927b3d6016928661afb002a"></audio>`

**Comment by petrvecera** [comment]
I don't think this is a problem of the library but rather issue in a shitty browser such as IE11?

**Comment by durdevic** [comment]
It's because of this. I have the same issue. Funnily somehow from iOS is posted it as a different file. Btw for me doesn't work on latest version on Yandex, and on Firefox I'm getting the wrong length of the file

https://developers.google.com/web/updates/2016/03/play-returns-promise?hl=en
